bindings/wrapper.h

```diff
@@ -1,6 +1,6 @@
 #ifndef __WRAPPER_H__
 #define __WRAPPER_H__
 
-#include <mysql/mysql.h>
+#include <mysql.h>
```

Comment:
What is the reasoning for this change? On my local system the header is in `/usr/include/mysql/mysql.h` so the old variant would pick it up automatically. The new variant would require an additional configuration. I would expect that to be the same on most linux distributions?

Reply:
The pkg-config is mean to set the `includedir`. This would be true in Linux and MacOS. The Windows also includes a single path, its the CI that goes out of the way to make mysql/mysql.h available.

```
cat /usr/lib/x86_64-linux-gnu/pkgconfig/mysqlclient.pc
# Copyright (c) 2015, 2023, Oracle and/or its affiliates.
...
prefix=/usr
includedir=${prefix}/include/mysql
libdir=${prefix}/lib/x86_64-linux-gnu

Name: mysqlclient
Description: MySQL client library
Version: 21.2.36
Cflags: -I${includedir} 
Libs: -L${libdir} -lmysqlclient
Libs.private: -lzstd -lresolv -lm
Requires.private: openssl
```

MariaDB:
```
#  cat /usr/lib/x86_64-linux-gnu/pkgconfig/libmariadb.pc
# 
#  pkg_config.pc.in
#
#  pkg_config configuration file 
#  For a detailed description of options, please visit
#  Dan Nicholson’s Guide to pkg-config (http://www.freedesktop.org/wiki/Software/pkg-config/)
#

prefix=/usr
includedir=${prefix}/include/mariadb/
libdir=${prefix}/lib/x86_64-linux-gnu/

Name: libmariadb
Version: 3.3.10
Description: MariaDB Connector/C dynamic library
Cflags: -I${includedir}
Libs: -L${libdir} -lmariadb
Libs.private: -lz -ldl -lm -lpthread -lssl -lcrypto
```

59c931a90687f861fba5d51f07a57fa8d2fd714e originally had `autogen_bindings(lib_dir, include_dir);` which is better than the `BINDGEN_EXTRA_CLANG_ARGS`.

https://docs.rs/pkg-config/latest/pkg_config/struct.Library.html#structfield.include_paths - 

Seems [clang_args](https://github.com/rust-lang/rust-bindgen/blob/main/bindgen/options/mod.rs#L1214) is still there. To take the includedir directly.

I think just need to make the build.rs use the includepath.

Reply:
Using the provided include path sounds like a good idea.